src/event_form/components/audience/form-audience.controller.js

```diff
@@ -26,7 +26,10 @@ function FormAudienceController(EventFormData, eventCrud, appConfig) {
   }
 
   function isBookableEvent() {
-    return EventFormData.getLocation().isDummyPlaceForEducationEvents;
+    if (EventFormData.getLocation().isDummyPlaceForEducationEvents) {
+      controller.audienceType = 'education';
```

Comment:
This function is used to perform a check but also has a side effect because of this change (setting the `audienceType` to `education`).

I'd suggest to change [this line](https://github.com/cultuurnet/udb3-angular/blob/fe29369ba4fec35d10773eee092d49654c598937/src/event_form/components/audience/form-audience.controller.js#L20) instead to:
`controller.audienceType = isBookableEvent() ? 'education' : EventFormData.audienceType;`